composer.json

```diff
@@ -13,13 +13,15 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3 | ^7.0",
-        "dhii/exception-interface": "^0.1"
+        "dhii/exception-interface": "^0.2",
```

Comment:
@mecha, when a newer version is available for upgrade, leave the old one as compatible, i.e. `^0.1|^0.2`. This will allow packages that depend on the same dependency at `0.1.x` to still use this.

Reply:
@XedinUnknown But this package is no longer compatible with `0.1.x` due to `dhii/factory-interface` requiring `^0.2`.

Reply:
That's the problem of `factory-interface`. *This* package requires `ThrowableInterface` only, which is present in `0.1`, which means it is compatible. Good catch WRT `factory-interface`; I'll fix that.

Reply:
Right. TBH, I didn't check if `factory-interface` was mistaken in requiring `^0.2`. I just assumed it has a reason to do so. This much, at least, is my bad.

Reply:
You are right in assuming that it's like that for a reason. But the deps of this package should reflect this package's requirements, not those of another package.